torch/_torch_docs.py

```diff
@@ -3361,6 +3361,27 @@ def merge_dicts(*dicts):
 Alias for :func:`torch.ge`.
 """)
 
+add_docstr(torch.gradient,
+           r"""
+gradient(input, *, domain=None, dim=None, edge_order=1) -> List of Tensors
+
+This function is analogous to numpy.gradient with minor changes in arguments.
+
+Here optional argument domain is provided as kwarg instead of varargs, and it can take values of
```

Comment:
I would remove this paragraph and add the structured elaboration of args and kwargs that other functions have. This should make it easier to elaborate on each kwarg.